mono/mini/exceptions-arm64.c

```diff
@@ -24,6 +24,12 @@
 
 #ifndef DISABLE_JIT
 
+#ifdef MONO_ARCH_ENABLE_PTRAUTH
+	static gboolean enable_ptrauth = TRUE;
```

Comment:
Why is a compile time constant converted to a runtime check? Is that to support cross compile cases?

Reply:
No good reason - I just copied it from https://github.com/Unity-Technologies/mono/blob/unity-main/mono/mini/mini-arm64.c#L278-L292